aiida_tbextraction/fp_run/wannier_input/pw2wann.py

```diff
@@ -0,0 +1,409 @@
+# -*- coding: utf-8 -*-
+
+# © 2017-2019, ETH Zurich, Institut für Theoretische Physik
+# Author: Dominik Gresch <greschd@gmx.ch>
+"""
+Defines a workflow that calculates the Wannier90 input files using Quantum ESPRESSO pw.x.
+"""
+
+import copy
+import gc
+import io
+import more_itertools
+import re
+import tempfile
+import os
+
+import numpy as np
+from w90utils.io import write_mmn
+
+from collections import ChainMap
```

Comment:
`collections` _is_ in the standard library, this belongs in the first block of imports.

Reply:
corrected in latest commit